css/style.css

```diff
@@ -0,0 +1,97 @@
+@media only screen and (max-width: 600px) {
+  body {
+    background-color: lightblue;
+    display: flex;
+  }
+}
+
+#navbar {
+  width: 100%;
+  background-color: brown;
+  position: fixed;
+  top: 0;
+}
+
+#navbar a {
+  float: right;
+  margin: 0 15px;
+  padding-bottom: 15px;
+  font-size: 1.8em;
+  color: rgb(255, 255, 255);
+  text-decoration: none;
+  text-align: center;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
```

Comment:
It's better to work using the class notion instead of IDs

Reply:
Optional: Good 👍 Job in pointing out about using a class but instead you can write "I would suggest you use more classes instead of IDs while writing your CSS code.".